lib/routes/concerts.js

```diff
@@ -0,0 +1,39 @@
+const { Router } = require("express");
+const router = Router();
+
+const { getConcert, getConcerts, postConcert } = require("../models/concerts");
+
+router.post(`/`, async (request, response) => {
+  try {
+    console.log(request);
+    console.log(request.body);
+    const createdConcert = await postConcert(request.body);
+    return response.json(createdConcert);
+  } catch (error) {
+    console.error(`Thats the error: ${error}`);
+    return response.end("Error");
```

Comment:
Maybe send a more detailed error message to your client (happens multiple times):
```suggestion
    return response.end(error.message);
```